provisioners/roles/postgres/tasks/main.yml

```diff
@@ -5,8 +5,28 @@
     dest=/etc/yum.repos.d/CentOS-Base.repo
   sudo: yes
 
+- name: install cert utilities
```

Comment:
This might be the only 1 of the 4 new tasks you need.  It worked for me with recent yum/https issues